scripts/build_base_industry_totals.py

```diff
@@ -117,8 +117,9 @@ def create_industry_base_totals(df):
     renaming_dit = transaction.set_index("Transaction")["clean_name"].to_dict()
     clean_industry_list = list(transaction.clean_name.unique())
 
+    # TODO Check with the sector-coupled modelling experts
     unsd_path = (
-        os.path.dirname(snakemake.input["energy_totals_base"]) + "/demand/unsd/data/"
+        os.path.dirname(snakemake.input["transactions_path"]) + "/demand/unsd/data/"
```

Comment:
Hello!
Apologies, the review of this PR may have been left in drafts.

The whole PR sounds great to me but I'm a bit in doubt about this line.
The rule build_base_energy_totals creates such folder with the target files here loaded.
I believe it would be advisable to add an output to the build base rule and an input here with the directory.
That should avoid the confusion and this inner hidden dependencies.
What do you think?